setup.py

```diff
@@ -45,4 +45,5 @@
     install_requires=REQUIRMENTS,
     packages=find_packages(exclude=("tests", "tests.*")),
     entry_points={"console_scripts": ["raven = src.cmdline:execute"]},
+    include_package_data=True,
```

Comment:
Is this required?
I tried to build the package without this, and it worked. Is there something I am missing?